pandas/core/indexes/multi.py

```diff
@@ -1412,6 +1412,45 @@ def from_product(cls, iterables, sortorder=None, names=None):
         labels = cartesian_product(labels)
         return MultiIndex(levels, labels, sortorder=sortorder, names=names)
 
+    @classmethod
+    def from_frame(cls, df, squeeze=True):
+        """
+        Make a MultiIndex from a dataframe
```

Comment:
Finish with period. Run `./scripts/validate_docstrings.py pandas.MultiIndex.from_frame` to make sure the docstring follows all the standards we validate.

Use `DataFrame` instead of dataframe.